### PR TITLE
move atorch and tfplus into independent projects

### PR DESCRIPTION
**Pull request by mingcheng**
reopen the closed pr #1359 

fyi: https://github.com/intelligent-machine-learning/dlrover/issues/1358 and the mentioned project has migrated to 

- https://github.com/intelligent-machine-learning/tfplus
- https://github.com/intelligent-machine-learning/atorch

So, this pr will clean the old codes ~~and import the library as git modules~~.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,8 +26,10 @@ and resource utilization.
 
 What's more, DLRover provides extension libraries of PyTorch and TensorFlow to Speed Up Training.
 
-- [ATorch](atorch/README.md): an extension library of PyTorch to Speed Up Training of Large LLM.
-- [TFPlus](tfplus/README.md): an extension library of TensorFlow to Speed Up Training of Search, Recommendation and Advertisement.
+- [ATorch](https://github.com/intelligent-machine-learning/atorch): an extension library of PyTorch to Speed Up Training of Large LLM.
+- [TFPlus](https://github.com/intelligent-machine-learning/tfplus): an extension library of TensorFlow to Speed Up Training of Search, Recommendation and Advertisement.
+
+UPDATE 20241127: ATorch and TFPlus have moved to the independent repositories and import as git submodules. so you shoud run `git submodule update --init --recursive` to get the latest code before you build the project.
```
Comment (skydoorkai) [inline README.md:32]: ATorch and TFPlus are independent python packages, and DLRover package build does not need these tow modules. Thus, it is not necessary to add them as submodules.

Comment (antfin-oss) [reply README.md:32]: Yep, +1

I recommend directly removing this part of the content.

Comment (mingcheng) [reply README.md]: OK, I updated the documents and removed the submodules by the following commits.

**Comment by workingloong** [comment]
Also remove atorch-python-test.

**Comment by mingcheng** [comment]
> Also remove atorch-python-test.

I have checked this test item and found codecov cached this case. I temporarily disable the app for the cache to clean by itself for a while.

**Review by workingloong — APPROVED** [review]
LGTM

**Comment by antfin-oss** [comment]
> > Also remove atorch-python-test.
> 
> I have checked this test item and found codecov cached this case. I temporarily disable the app for the cache to clean by itself for a while.

This is a configuration issue. fixed

**Review by BalaBalaYi — APPROVED** [review]

**Review by majieyue — APPROVED** [review]
LGTM

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -27,7 +29,7 @@ jobs:
       - name: Upload coverage reports to Codecov
         uses: codecov/codecov-action@v4.0.1
         with:
-          token: 043586e5-68b8-4588-9bf4-c333f2692345
```
Comment (antfin-oss) [inline .github/workflows/main.yml:30]: Does this code token need to be removed?

**Review by skydoorkai — APPROVED** [review]
lgtm

**Comment by mingcheng** [comment]
updated the commit history and removed some unnecessary or redundant messages.

**Review by nash635 — APPROVED** [review]

**Pull request merged**